src/firstscreen.cpp

```diff
@@ -28,7 +28,7 @@ FirstScreen::FirstScreen( PackageBackend *backend, QString *tmpFileName, const Q
     QVBoxLayout *mainLayout = new QVBoxLayout;
     mainLayout->setSpacing( 0 );
 
-    QFile dataFile( m_tmpFileName->toAscii() );
+    QFile dataFile( m_tmpFileName->toLatin1() );
```

Comment:
Cause Latin1 and Ascii encodings have a bit more in common than with UTF. Changing it to UTF ( toUtf8() ) is not a problem though.